prow/repoowners/repoowners.go

```diff
@@ -736,8 +737,35 @@ func (o *RepoOwners) applyOptionsToPath(path string, opts dirOptions) {
 	}
 }
 
-func (o *RepoOwners) filterCollaborators(toKeep []github.User) *RepoOwners {
-	collabs := sets.New[string]()
+// Extend github team to appproves/reviewers list
+// If appproves/reviewers name contains '/', like 'myorg/developer'
```

Comment:
IMHO I don't like the idea of treating the strings that contain a `/` char as a GitHub team. I prefer a different section in the owner's configuration for the GitHub teams instead. @petr-muller WDYT?

Reply:
No strong opinion. I'm fine with it. We use _"contains `/`"_ to distinguish `org/repo` from `org` strings in other places, I think it is fairly pragmatic :shrug: